beacontools/scanner.py

```diff
@@ -108,7 +103,7 @@ def run(self):
         self.socket.close()
 
     def set_scan_parameters(self, scan_type=ScanType.ACTIVE, interval_ms=10, window_ms=10,
-                            address_type=BluetoothAddressType.RANDOM, filter_type=ScanFilter.ALL):
+                            address_type=BluetoothAddressType.PUBLIC, filter_type=ScanFilter.ALL):
```

Comment:
I don't like changing this globally because it has privacy implications. Scanning should be done using randomized MACs to avoid the possibility of tracking. If this really fixes #43 we should rather add a parameter to `BeaconScanner.__init__` so that users who need this option can set it.